docs/concepts/feature-service.md

```diff
@@ -0,0 +1,29 @@
+# Feature Service
+
+### Feature Service
+
+A feature service is an object that represents a logical group of features from one or more [feature views](feature-view.md). 
+Feature Services allows features from within a feature view to be used as needed by an ML model. Users can expect to create one Feature Service per Model, keeping the features needed by the model tracked.   
+
+{% tabs %}
+{% tab title="driver\_trips\_feature\_service.py" %}
+```python
+from driver_trips_feature_view import driver_stats_fv
+
+driver_stats_fs = FeatureService(
+    name="driver_activity",
+    features=[driver_stats_fv]
```

Comment:
We should demonstrate multiple feature views being used here

Reply:
Done